### PR TITLE
[peertube] Extract files also from `streamingPlaylists`

### PR DESCRIPTION
**Pull request by 0l-l0**
Another pull request in continuation of #26302. Re-requested due to DMCA take-down.

### CODE REVIEW

File: youtube_dl/extractor/peertube.py [hunk 1]
```diff
@@ -526,7 +538,15 @@ def _real_extract(self, url):
         title = video['name']
 
         formats = []
-        for file_ in video['files']:
+        files = video.get('files') or []
+        for playlist in (video.get('streamingPlaylists') or []):
+            if not isinstance(playlist, dict):
+                continue
+            playlist_files = playlist.get('files')
+            if not playlist_files or not isinstance(playlist_files, list):
```
Comment (remitamine) [inline youtube_dl/extractor/peertube.py:546]:
```suggestion
            if not (playlist_files and isinstance(playlist_files, list)):
```

**Comment by aandroide** [comment]
after this update it no longer recognizes the url. up to 15 hours ago everything worked fine with the January 3 release. please fix this pullover

**Comment by aandroide** [comment]
already 3 people have opened a problem after this update.

**Comment by aandroide** [comment]
I apologize and reticulate a reboot was enough

**Pull request merged**